index.html

```diff
@@ -36,6 +36,9 @@
 						<a href=""><i class="material-icons uk-text-inverse">search</i></a>
 						<input type="text" name="search" value="" class="uk-input uk-width-medium" placeholder="Search…" uk-dropdown="animation: uk-animation-slide-right-small; pos: left-center; offset: 0; delay-hide:100; mode: click;">
 					</li>
+					<li>
+						<div id="notifications"></div>
```

Comment:
to discuss later: should this be a generic "put your icon there" area for other apps to use?

Reply:
Phoenix will spawn and supply a DOM node for the render function.